test/extended/README.md

```diff
@@ -33,6 +36,67 @@ $ go get github.com/onsi/ginkgo/ginkgo
 You also need to have the `openshift` binary in the `PATH` if you want to use
 the shell script helpers to execute the extended tests.
 
+Rapid local testing
+--------------------
+
+If you already have a running OpenShift cluster, e.g. one created using `oc
+cluster up`, you can skip having the extended test infrastructure spin up an
+OpenShift cluster each time the tests are run by setting the `TEST_ONLY`
+environment variable as follows:
+
+```console
+$ oc cluster up
+$ oc login -u system:admin
+$ export KUBECONFIG=${KUBECONFIG-$HOME/.kube/config}
+```
+
+Then, for example:
+```console
+$ make build-extended-test
+$ TEST_ONLY=1 test/extended/core.sh --ginkgo.focus='\[builds\]'
+```
+
+By default the Kubernetes test framework will remove the project associated with
+your test spec when it completes, regardless of whether it fails or not.  This
+can be inconvenient when debugging.  To stop this behaviour, add the argument
+--delete-namespace=false:
```

Comment:
I was debugging a test where a failed pod was immediately deleted -- perhaps it was just a per-test thing.